src/main/java/Player.java

```diff
@@ -1,7 +1,7 @@
 import java.util.List;
 public class Player {
     public String name;
-    private char color;
+    private char color; // 'w' or 'b'
```

Comment:
David has previously used capital 'W' or 'B'. If you change this, please notify the team so that it can be standardized and documented in the Player class.